package.json

```diff
@@ -46,6 +46,7 @@
     "@types/lodash": "^4.14.178",
     "@types/node": "^16.11.12",
     "@types/react": "^17.0.37",
+    "@types/recharts": "^1.8.23",
```

Comment:
uh. yeah. it is big.. it's popular and easy to use 🤷

Reply:
🤣 

+20% bundle size! That's crazy!

A quick look and it is tree shakeable.. just need some effort into doing it properly and stuff.. I doubt it'll be a problem once `common` is being shaked